File-Formats/DocIO/Working-with-Charts.md

```diff
@@ -467,6 +467,170 @@ document.Close()
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Charts/Create-custom-chart).
 
+## Creating a Chart from Database
+
+Generate a chart within a Word document sourced from a database by retrieving the required chart data values and subsequently assigning them to the chart's dataset.
```

Comment:
Create a chart in a Word document using the .NET Word Library by utilizing the values retrieved from the database.